pytorch/training/docker/1.11/py3/cu115/Dockerfile.gpu

```diff
@@ -480,6 +480,13 @@ RUN pip install --no-cache-dir -U \
  && pip uninstall -y torch \
  && pip install --no-cache-dir -U ${PT_TRAINING_URL}
 
+# Install Studio Notebook Utils
```

Comment:
have you tested that the cu115 images build successfully? i don't think they are enabled in the buildspec